ext/pdo/pdo_stmt.c

```diff
@@ -40,6 +40,12 @@
 	  RETURN_FALSE;	\
   }	\
 
+#define PHP_STMT_GET_OBJ_INT	\
```

Comment:
We should be throwing an exception in this case, right? (in `PHP_STMT_GET_OBJ ` as well)